image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -12,6 +12,7 @@ metadata:
     {{- include "srox.labels" (list . "deployment" "scanner") | nindent 4 }}
   annotations:
     {{- include "srox.annotations" (list . "deployment" "scanner") | nindent 4 }}
+    openshift.io/required-scc: nonroot-v2
```

Comment:
Would it make sense to put it under condition?
```
{{- if ._rox.env.openshift }}
```

Reply:
We could do that, yes.
I opted to include it always as I wouldn't expect this annotation to interfere with normal operations outside of OpenShift.
Thinking about it, it would probably be cleaner if we would do that only on OpenShift.

Reply:
Done!